app/models/portfolio.rb

```diff
@@ -24,6 +24,7 @@ def add_portfolio_item(portfolio_item)
   def metadata
     user = UserContext.new(Insights::API::Common::Request.current!, nil)
 
-    {:user_capabilities => PortfolioPolicy.new(user, self).user_capabilities}
+    {:user_capabilities => PortfolioPolicy.new(user, self).user_capabilities,
+     :shared            => self.access_control_entries.any?}
```

Comment:
May be off topic, but `:shared` seems close to what we were thinking with a potential `:orderable` metadata key.  Would largely be handled like #612 except we would pull out the `/orderable` endpoint.

Reply:
@syncrou This a UI eye candy, if the Portfolio has been shared with other users you can put a shared icon in the tile. Not sure if you have seen how windows shows shared items in the UI.

Reply:
I think that's similar to what #612 is aiming to solve, except it's a separate endpoint. If we included another `:orderable` key in the metadata here, the UI could grey out the order button or put text saying "hey this thing isn't orderable right now because of X reason", or whatever.

I think it's a fine idea in theory, just not sure how well it would work when fetching an index and having to potentially call topo a bunch like how we were calling RBAC a bunch.

Reply:
@eclarizio yes. @mkanoor  I should have better clarified it as another attribute in metadata.  @eclarizio's point about performance concerns makes a lot of sense.  

We'll probably table action on #612 until your current performance issues are handled, and we could potentially piggy back off your solutions.